pybandits/quantitative_model.py

```
@@ -1148,3 +1148,297 @@ class CmabZoomingModelCC(BaseCmabZoomingModel, QuantitativeModelCC):
+            Each element can be a float (for 1D quantities) or a list (for multi-dimensional).
+        context : ArrayLike
+            The context value(s) associated with each observation.
+
+        Returns
+        -------
+        np.ndarray
+            The input for the network, concatenated quantity and context.
+        """
+        if isinstance(quantity[0], np.ndarray):
+            if quantity[0].ndim != 1:
+                raise TypeError("Quantity must be a 1D array")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)